composer.json

```diff
@@ -21,15 +21,15 @@
     },
     "conflict": {
         "doctrine/dbal": "<3.5 || >=5",
-        "doctrine/orm": "<2.14 || >=4",
+        "doctrine/orm": "<3.0 || >=4",
         "doctrine/phpcr-odm": "<1.3.0"
     },
     "require-dev": {
         "ext-sqlite3": "*",
         "doctrine/coding-standard": "^12",
-        "doctrine/dbal": "^3.5 || ^4",
+        "doctrine/dbal": "^3.7.3 || ^4",
```

Comment:
This version constraint bump was necessary to avoid

```
1) Doctrine\Tests\Common\DataFixtures\ORMExecutorSharedFixtureTest::testSharedFixtures
Doctrine\ORM\Tools\ToolsException: Schema-Tool failed with Error 'An exception occurred while executing a query: near ".": syntax error' while executing DDL: CREATE INDEX IDX_3155D2E5F675F31B ON readers.author_reader (author_id)

/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Driver/API/SQLite/ExceptionConverter.php:68
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1938
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1880
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1208
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/orm/src/Tools/SchemaTool.php:81
/home/greg/dev/doctrine-data-fixtures/tests/Common/DataFixtures/Executor/ORMExecutorSharedFixtureTest.php:47

Caused by
Doctrine\DBAL\Exception\SyntaxErrorException: An exception occurred while executing a query: near ".": syntax error

/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Driver/SQLite3/Exception.php:16
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Driver/SQLite3/Connection.php:59
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1206
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/orm/src/Tools/SchemaTool.php:81
/home/greg/dev/doctrine-data-fixtures/tests/Common/DataFixtures/Executor/ORMExecutorSharedFixtureTest.php:47

Caused by
Doctrine\DBAL\Driver\SQLite3\Exception: near ".": syntax error

/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Driver/SQLite3/Connection.php:57
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1206
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/orm/src/Tools/SchemaTool.php:81
/home/greg/dev/doctrine-data-fixtures/tests/Common/DataFixtures/Executor/ORMExecutorSharedFixtureTest.php:47

Caused by
SQLite3Exception: near ".": syntax error

/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Driver/SQLite3/Connection.php:57
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/dbal/src/Connection.php:1206
/home/greg/dev/doctrine-data-fixtures/vendor/doctrine/orm/src/Tools/SchemaTool.php:81
/home/greg/dev/doctrine-data-fixtures/tests/Common/DataFixtures/Executor/ORMExecutorSharedFixtureTest.php:47
```

Reply:
That failure was fixed in https://github.com/doctrine/dbal/pull/6226, which was prompted by… https://github.com/doctrine/data-fixtures/pull/452 :thinking: :thinking: :thinking:

Reply:
It's a major release and the version bump isn't that big. :+1:

Reply:
Anyway, I reverted it with the ORM bump. It will come back at a later date I guess.